### ISSUE TITLE
About comparison with Kalantari's method

### ISSUE THREAD

**Issue by Rhysess** [guanyingc/DeepHDRVideo#9]
Dear Guanying,

I have a question about the comparative experiment, and I hope you could give some suggestions.

You give results with your method and Kalantari19's method to compare. However, in Kalantari19's paper they said "
use mini-batches of size 10 and perform training for 60,000 iterations" without specifying concrete number of epochs. So I wonder how you train their method and how epochs you use for training Kalantari19's with your dataset.

Looking forward to your reply. Thanks.

**Comment by guanyingc** [comment]
Hi, I forget about the extact setup of training Kalantari19's method. If I remember it correctly, I use an epoch number like 15 or 20. In fact, the results may not change a lot after 15 epochs. You might monitor the result on validation set to find a good stop point.

**Comment by Rhysess** [comment]
Thank you very much, Guanying! Your suggestion is very helpful. I will close this issue.
